USE-CASES/use-case-vertical-eclass-retail.html

```diff
@@ -0,0 +1,139 @@
+IoT not only supports the increase in efficiency through general data exchange in the networking among each other. IoT also enables the procurement of new items as well as spare parts. A detailed product description in online plattforms and the device itself makes it possible to search the IoT for required or equivalent components.<br>
+The Web of Things may help to exchange and look up the relevant product classification as well as product description.
+
+</dd>
+<dt>Expected Devices</dt>
+<dd>
+Each device from each domain.
+
+</dd>
+<dt>Expected Data</dt>
+<dd>
+To cover the case of retail as well as purchasing and sales, product master data has to be exchanged. 
+
+</dd>
+<dt>Dependencies - Affected WoT deliverables and/or work items</dt>
+<dd>
+
+&lt;List the affected WoT deliverables that have to be changed to enable this use case&gt;
+Thing Model: Define a standardized way to bind standardized product master data including a structured form.
+
+</dd>
+<dt>Description</dt>
+<dd>
+
+Device trading is a crucial and economic factor in our daily lives. Automated trading in particular will increase in the context of IoT. In order for products to be findable, they must be well described. In order to compare products, the product descriptions must be uniform. At the very least, the same classification systems must be used. Especially in automated commerce, the machine interpretability of used properties is of high importance. This applies to the classical new acquisition of complete devices, components and also services from catalogs. But also the search and find of items already on the market or find needed services.
+
+<p>#### Variants:</p>
+
+Possible use case variants are:
+<ul>
+<li>Initial purchase incl. search and find in online shops</li>
+<li>Automatic spare part procurement</li>
+<li>Finding available components that can be used temporary. These can be rented at short notice, for example.</li>
+<li>Service provision</li>
+<li>Offering a specific function through a device which can be soled</li>
+</ul>
+
+</dd>
+<dt>Security Considerations</dt>
+<dd>
+none
+
+</dd>
+<dt>Privacy Considerations</dt>
+<dd>
+none
+
+</dd>
+<dt>Accessibility Considerations</dt>
+<dd>
+none
+
+</dd>
+<dt>Internationalisation (i18n) Considerations</dt>
+<dd>
+Internationalization is a very importnat toipic in this use case. Data must initially be maintained by humans. For this purpose, corresponding translations for characteristics and classes must be available, which are used to create the master data and to interpret the data. But also a machine-interpretable evaluation of the values is important.
+
+</dd>
+<dt>Requirements</dt>
+<dd>
+It is necessary to define a common way to define retail data in the WoT Thing Description or define Templates for each asset Type.
+
+</dd>
+<dt>Gaps</dt>
+<dd>
+There is no standardized semantic and vocabulary for product meta and master data as well as the InteractionAffordance.
+
+</dd>
+<dt>Existing standards</dt>
+<dd>
+
```

Comment:
Other standards should be mentioned as well :
- GS1
- W3C data set exchange
- 
Mapping of classifications across description schemes